src/guides/protocol.md

```diff
@@ -0,0 +1,271 @@
+## Handshake Protocol Summary
+
+This guide is intended for developers with a working knowledge of the Bitcoin
+protocol and its general consensus parameters. All values below describe mainnet
+parameters.
+[Testnet, regtest, and simnet may differ.](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js)
+Links reference the `hsd` codebase at the time of writing, during Testnet 4.
+
+### Blockchain
+
+- Block interval time:
+[10 minutes](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js#L166)
+
+- PoW Difficulty adjustment interval: [144 Blocks (one day)](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js#L158-L159)
```

Comment:
PoW actually retargets every single block, but it's BASED on the last 144 blocks as a target for one day of time.